pkg/endpoint/log.go

```diff
@@ -41,6 +49,12 @@ func (e *Endpoint) getLogger() *logrus.Entry {
 	return (*logrus.Entry)(v)
 }
 
+// getPolicyLogger returns a logger to be used for policy updates, or nil, if not configured.
```

Comment:
Would be good to ensure that we make it clear when this should be used, ie in debugging code:

```suggestion
// getPolicyLogger returns a logger to be used for policy update debugging, or nil, if not configured.
```

Reply:
Fixed.